lib/openers/evince-opener.js

```diff
@@ -19,8 +19,8 @@ const DBUS_NAMES = {
 
 function syncSource (uri, point) {
   const filePath = decodeURI(url.parse(uri).pathname)
-  atom.workspace.open(filePath).then(editor => editor.setCursorBufferPosition(point))
   atom.focus()
+  atom.workspace.open(filePath).then(editor => editor.setCursorBufferPosition(point))
```

Comment:
Why do you focus first and then sync? I tried out both and either seems to work. But me personally feel that syncing first and then focus feels a bit more natural. Is there a technical motivation behind it?
(In the case of first focus and sync I could sometimes see the content of my editor windows move around.)

PS: This is so awesome.